tardis/_astropy_init.py

```diff
@@ -122,7 +122,7 @@ def test(package=None, test_path=None, args=None, plugins=None,
     if not os.environ.get('ASTROPY_SKIP_CONFIG_UPDATE', False):
         config_dir = os.path.dirname(__file__)
         try:
-            config.configuration.update_default_config(__package__, config_dir)
+            config.configuration.update_default_config(__package__, config_dir, version=__version__)
```

Comment:
Weird. I don't think this should be necessary, but I haven't really looked closely at the problem yet.  You should be getting a `version.py` generated when you run `setup.py`.